.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+---
+name: Generator tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    container: golang:1.13
+    steps:
+      - uses: actions/checkout@v2
+      - run: go test -p 1 ./...
```

Comment:
Might want a actions/setup-go@v2 and pin a Go version. I am not sure what we be used otherwise.

Reply:
Good idea, much clearer. Done.